statsmodels/tsa/tests/test_arima.py

```diff
@@ -2237,6 +2237,16 @@ def test_arima_fit_mutliple_calls():
         mod.fit(disp=0, start_params=[np.mean(y), .1, .1, .1])
     assert_equal(mod.exog_names,  ['const', 'ar.L1.y', 'ma.L1.y', 'ma.L2.y'])
 
+    #test multiple calls when there is only a constant term
+    mod = ARIMA(y, (0, 0, 0))
+    # Make multiple calls to fit
+    with warnings.catch_warnings(record=True) as w:
+        mod.fit(disp=0, start_params=[np.mean(y)])
+    assert_equal(mod.exog_names,  ['const'])
+    with warnings.catch_warnings(record=True) as w:
+        mod.fit(disp=0, start_params=[np.mean(y)])
+    assert_equal(mod.exog_names,  ['const'])
+
```

Comment:
you can add here a "smoke test" for summary()
i.e. assign the last fit to `res` and add aline
res.summary()

We don't have non-smoke test for most of the summary calls, because content is not stable across runs